data/naltok outfits.txt

```diff
@@ -376,4 +376,4 @@ outfit "Ministry of War License"
 outfit "Ministry of Transport License"
 	category "Licenses"
 	thumbnail "outfit/ministry transport license"
-	description "While most ships are made readily available by the Telashki, the mass-produced transport vessels that form the backbone of their economy are generally only used by the Ministry of Transport."
+	description "While most ships are made readily available by the Telashki, the mass-produced transport vessels that form the backbone of their economy are generally only used by the Ministry of Transport, and not sold in many places. This license lets you buy them, if you can find a place to."
```

Comment:
They're actually sold pretty widely if you have the license - there's only one shipyard in the whole of Telashki space that _doesn't_ sell the Conger, though the Marlin is a little rarer. Clarifying in the license description that the license lets you buy them does make sense though; I might expand on that once I've decided how you actually get this license.